nemo/collections/asr/metrics/rnnt_wer_bpe.py

```diff
@@ -255,12 +255,5 @@ def compute(self):
 
 
 @dataclass
-class RNNTBPEDecodingConfig:
-    strategy: str = "greedy_batch"
-    compute_hypothesis_token_set: bool = False
-
-    # greedy decoding config
-    greedy: greedy_decode.GreedyRNNTInferConfig = greedy_decode.GreedyRNNTInferConfig()
-
-    # beam decoding config
-    beam: beam_decode.BeamRNNTInferConfig = beam_decode.BeamRNNTInferConfig(beam_size=4)
+class RNNTBPEDecodingConfig(RNNTDecodingConfig):
```

Comment:
Do we need this class anymore?

Reply:
Well.. lets keep it. Subword decoding has till now thankfully remained 1:1 with char decoding at the API level and I plan on keeping it that way for as long as possible, but who knows, it may diverge in the future.